lib/messages/create/comment-created.js

```diff
@@ -0,0 +1,17 @@
+const {
+  Message,
+} = require('..');
+
+module.exports = class CommentCreated extends Message {
+  constructor(commentUrl) {
+    super({});
+    this.commentUrl = commentUrl;
+  }
+
+  getAttachment() {
+    return {
+      ...super.getBaseMessage(),
+      text: `:white_check_mark: Comment successfully created: ${this.commentUrl}`,
```

Comment:
Would it make sense to do this more in the form of the other activity messages where it's `${author} commented` and then the title of the issue/PR?

Reply:
Yeah, we could do that. Would we do this as an `in_channel` or `ephemeral` message? 

I suppose `in_channel` could be confusing to other users in the channel who did not invoke the message action. 

We should also only post this message using logic in this module if the repo and channel are not subscribed to comments anyway

Reply:
Just looked into this, and unfortunately we don't get the issue/pr title or repository information from the create comment endpoint: https://developer.github.com/v3/issues/comments/#create-a-comment

We'd need that information to re-use the existing rendering logic. We don't really have an opportunity to make additional API calls before the dialog times out. A queue system would help here.